include/Game/Cave/RandMapMgr.h

```diff
@@ -177,7 +177,7 @@ struct RandMapMgr : public CNode {
 	ObjectLayoutInfo* makeObjectLayoutInfo(int placedNodeIndex);
 	void getStartPosition(Vector3f& position, int index);
 	void getItemDropPosition(Vector3f& position, f32 minDist, f32 maxDist);
-	void getItemDropPosition(Vector3f* positions, int positionCount, f32 lowerWeightBound, f32 upperWeightBound);
+	void getItemDropPosition(Vector3f positions[], int positionCount, f32 lowerWeightBound, f32 upperWeightBound);
```

Comment:
nit: we don't use array notation for any parameters elsewhere in the codebase, so probably good to keep this in pointer notation - if we want to, we can do a pass through the codebase later and make any lists into array notation to make it clearer later.